### DISCUSSION TITLE
External Help

### DISCUSSION THREAD

**Discussion by jdhitsolutions** [SamErde/PSPreworkout#21] [General]
It looks like you've used Platyps to generate intermediate markdown files. Do you intend to create external help and ship it with the module? Once you do that, you can delete the comment-based help from the functions.

**Comment by SamErde** [comment]
I have not yet tackled the question of how to setup external help but it is on my list. Are there advantages to using external help and removing comment based help? Smaller/faster download?

**Reply by jdhitsolutions** [reply]
Having external help I think is a little faster. More professional and has the option of being localized. I think it is a good practice.

**Comment by SamErde** [comment]
Working through this one still as I learn more about PlatyPS and external help in PowerShell. Sean Wheeler provided this great resource: <https://sdwheeler.github.io/seanonit/docs/06-pshelp/>.
Created a new GitHub action that makes .zip and .cab files of the markdown help from the XML.
Added **HelpInfoURI** to the manifest.

**Comment by SamErde** [comment]
I do believe this is now complete!
